### PR TITLE
Improving commit subject clarity for pull request builds

### PR DESCRIPTION
**Pull request by blackjackkent**
First of all, thank you for this webhook! Saved me a ton of work and is very clean. One issue I did encounter, however, is that when the build is a pull request build, the commit subject is a little obtuse...

image

This update makes use of the `$env:APPVEYOR_PULL_REQUEST_TITLE` environment variable that is available during PR builds to produce a more friendly message in this scenario:

image

### CODE REVIEW

File: send.ps1 [hunk 1]
```diff
@@ -48,6 +48,7 @@ else {
 }
 
 if ($env:APPVEYOR_PULL_REQUEST_NUMBER) {
+  $COMMIT_SUBJECT="Pull Request #$env:APPVEYOR_PULL_REQUEST_NUMBER (from $env:APPVEYOR_PULL_REQUEST_HEAD_REPO_NAME): $env:APPVEYOR_PULL_REQUEST_TITLE"
```
Comment (iamtraction) [inline send.ps1]: This is too long. How about not including where the PR came from?

Probably something like this:
```suggestion
  $COMMIT_SUBJECT="PR #$env:APPVEYOR_PULL_REQUEST_NUMBER: $env:APPVEYOR_PULL_REQUEST_TITLE"
```

**Review by iamtraction — CHANGES_REQUESTED** [review]
Great PR, but just one suggestion.

**Comment by blackjackkent** [comment]
(Oops, didn't realize I could directly apply your suggestion through Github until I'd already pushed.)

Makes sense to me! Updated the branch accordingly.

**Review by iamtraction — APPROVED** [review]
👌

**Comment by iamtraction** [comment]
Thank you for your contribution! We would like to thank you by **inviting you to join the DiscordHooks organization on GitHub.** Accept the invite by visiting https://github.com/orgs/DiscordHooks/invitation.

Thanks again!

**Pull request merged**